File: hello/witaj.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  witaj.py
#  
#  
ROK_TERAZ = 2017
ROK_PYTHON = 1991

def parzyste(n):
    ile = list(range(0, n+1, 2))
    print(ile)
    return len(ile)

def main(args):
    
    imie= input("Jak sie nazywasz? ")
    print ("Witaj") , imie
    
    wiek = int(input("Ile masz lat?"))
    rok_urodzenia = ROK_TERAZ
    print("Rok urodzenia: ", 2017 - wiek)
    
    if rok_urodzenia < ROK_PYTHON:
        print("Jestem mlodszy")
    elif rok_urodzenia > ROK_PYTHON:
        print("Jestem starszy")
    else:
        print("Mamy tyle samo lat!")
    
    n = int (input("Podaj liczbę: "))
    print("Parzyste: ", parzyste(n))
    
    
    return 0

if __name__ == '__main__':
    
    
    import sys
    sys.exit(main(sys.argv))

File: hello/witaj.cpp
/*
 * witaj.cpp

 */


#include <iostream>

using namespace std;

int parzyste(int n)
{
    int i;
    for (i=0; i <=n; i+=2)
        cout<< i << " ";
    return i/2; 
}

int main(int argc, char **argv)
{
    int n;
    
    char tekst[20];
    cout <<"Witaj. Jak się nazywasz?";
    cin >> tekst;
    cout <<"Witaj" << tekst << endl;
    
    int lata;
    cout<< "Hej. Ile masz lat?" << endl;
    cin >> lata;
    cout <<"Urodziłeś się w " << 2017 - lata << endl;
    
    int data;
    data = 1983;
    if( data < 2017 - lata) 
    { cout <<"Jestem starszy :)"<< endl;
    }
    if( data > 2017 - lata) 
    { cout << "Jestem młodszy"<< endl;
	}
    if( data == 2017 - lata)
    { cout << "Mamy tyle samo lat!"<< endl;
        }
        
    
    cout << "Podaj liczbę: ";
    cin >> n;
    cout << "Parzystych: " << parzyste(n);
    
    return 0;
}

